ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -2634,6 +2634,7 @@ tests:
     cluster_profile: nutanix-qe
     env:
       E2E_RUN_TAGS: '@amd64 and @nutanix-ipi and @network-ovnkubernetes and not @fips'
+      SIZE_VARIANT: compact
```

Comment:
You to combine `compact` with `boot-categories-project`? If yes, you need to update the test job name accordingly to reflect that.

Reply:
Or maybe we can create a new dedicated test job for compact cluster?